packages/jamf_protect/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.5.0"
+  changes:
+    - description: Added host.name field, added "Security Solution" tag to Telemetry dashboard and modified the Jamf Protect Alerts dashboard to have some additional filters applied to not show Telemetry data, also changed start_time in authentication pipeline.
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/10931
```

Let's break this up to improve readability.

```suggestion
    - description: Added a lowercased host.name field to the telemetry data stream.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/10931
    - description: Added a "Security Solution" tag to the Telemetry dashboard.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/10931
    - description: Modified the Jamf Protect Alerts dashboard to include additional filters and exclude Telemetry data.
      type: enhancement
      link: https://github.com/elastic/integrations/pull/10931
    - description: Fixed handling of process.start in the telemetry data stream pipeline.
      type: bugfix
      link: https://github.com/elastic/integrations/pull/10931
```

---

Oh learned something new today! did not knew this was possible!